tensorflow/examples/image_retraining/retrain.py

```diff
@@ -450,7 +452,14 @@ def get_or_create_bottleneck(sess, image_lists, label_name, index, image_dir,
                                 category)
     if not gfile.Exists(image_path):
       tf.logging.fatal('File does not exist %s', image_path)
-    image_data = gfile.FastGFile(image_path, 'rb').read()
+    jpgext = ['jpg', 'jpeg', 'JPG', 'JPEG']
```

Comment:
Make this a separate helper method that takes in file_name and returns back image_data.